README.md

```diff
@@ -4,8 +4,8 @@ CLI for [cats](http://imcs.dvfu.ru/cats). Currently only for jury
 
 - Install [python3](https://python.org)
 - Install [git](https://git-scm.com/)
-- Install [requests](http://docs.python-requests.org/en/latest/) python package with pip
 - Clone this repository
```

Comment:
nice to explicitly specify `git clone` command too

I suggest You to change this part as following:
- Install [python3](https://python.org) and [git](https://git-scm.com/)
- Clone repo, install requirements, add cats.py to your PATH
```bash
git clone https://github.com/FooBarrior/cats-cli
pip install -r requirements.txt
ln -s cats.py ~/.local/bin/cats
```